migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -107,14 +99,13 @@ include::modules/migration-understanding-cpma.adoc[leveloffset=+2]
 include::modules/migration-installing-cpma.adoc[leveloffset=+2]
 include::modules/migration-using-cpma.adoc[leveloffset=+2]
 
-== Troubleshooting a failed migration
+== Troubleshooting
 
 You can view the migration custom resources (CRs) and download logs to troubleshoot a failed migration.
 
 include::modules/migration-custom-resources.adoc[leveloffset=+2]
 include::modules/migration-viewing-migration-crs.adoc[leveloffset=+2]
 include::modules/migration-downloading-logs.adoc[leveloffset=+2]
 include::modules/migration-restic-timeout.adoc[leveloffset=+2]
-
-include::modules/migration-known-issues.adoc[leveloffset=+1]
-:ocp-3-to-4_2!:
+include::modules/migration-known-issues.adoc[leveloffset=+2]
+:migrating-3-4!:
```

Comment:
I'm not sure what this attribute (`:migrating-3-4!:`) does. Looks like with `!`, you don't use `:context:` but without it, you do. I'm sure what you have is right, and that it has something to do with what version you are migrating from, so just pointing it out in case something is missing.

Reply:
It's OK. There are several changes of context. This `:migrating-3-4!:` isn't absolutely necessary, but it's neater that way. It turns off the `:migrating-3-4:` context, nothing to do with whether you use it with `:context:` or not. :-)